src/sparseml/exporters/transforms/remove_duplicate_qconv_weights.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+
+import numpy
+from onnx import ModelProto, numpy_helper
+
+from sparseml.exporters.transforms.onnx_transform import OnnxTransform
+from sparseml.onnx.utils import ONNXGraph
+
+
+__all__ = ["RemoveDuplicateQConvWeights"]
+
+
+class RemoveDuplicateQConvWeights(OnnxTransform):
+    """
+    Removes weights of QLinearConv and ConvInteger that are the same.
```

Comment:
nit: "removes" -> "deduplicates" or something that doesn't imply that we're deleting all